apps/api/urls.py

```diff
@@ -6,5 +6,5 @@
 
 urlpatterns = [
     path("experiments/", views.get_experiments, name="list-experiments"),
-    path("participants/experiments/data", views.update_participant_data, name="update-participant-data"),
+    path("participants/<uuid:participant_id>", views.update_participant_data, name="update-participant-data"),
```

Comment:
this won't always be a UUID will it?

Reply:
Oh sorry forgot to update this name. This is now the participant's `public_id`, which is always a UUID, buuuuuut I'm now remembering that CommCare wouldn't know the participant's public id if we consider the workflow for SUDCare

Reply:
CommCare 'could' know the UUID if it's returned when CC creates the participant (assuming that's what happens). Otherwise we could accept either or require users to do a 'search' using the identifier and then use the public ID for writes.

Reply:
Yeah depends on the final workflow, but if I remember correctly we'll require the participant to make first contact, so we wouldn't need to "register" a participant in the traditional sense (that is, CC makes a request to OCS which creates the participant in the DB). My understanding of this registration is more to do with adding data to their record (for now)

Reply:
Either way, I think we can leave this as the public ID for now. What do you think of adding the the participant public ID and experiment session public ID to the message response?

Otherwise we'd need to provide an API to access the participant public ID for a user (thinking of Brian doing testing).

Reply:
I think if we're going to include those details, let's maybe instead have a GET "participants/<identifier>/info" endpoint. It feels too random to include it in the response of the update data call. We can then use this to get various info + stats (eventually, maybe) of the participant. So CC would be able to use this to get the `public_id` and in turn use that for the update_data call

Reply:
Ok so I sort of went the other way and just allowed the user to specify the participant identifier in the url. No need to do an additional request to get the public id. We _can_ still add an endnpoint to get participant data (latest session id, public_id etc), but since we don't really have a usecase for that right now, I think let's maybe leave it until we need it? Sorry for the back-and-forth on the public_id/identifier.